.github/workflows/stale.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Close stale PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          debug-only: true
+          skip-stale-issue-message: true
+          stale-pr-message: 'Message to comment on stale PRs. If none provided, will not mark PRs stale'
+          days-before-stale: 60
+          days-before-close: 7
+          days-before-issue-stale: -1
```

Comment:
Should we enable it for issues as well? There are probably lots of outdated issues that nobody is gonna look at. But maybe give a bigger number to become stale.. like `360`?